src/api/bank-accounts/bank-accounts.md

```diff
@@ -275,6 +275,7 @@ for bank accounts.
     "verified": false,
     "directDebitAuthorized": true,
     "createdAt": "2020-06-12T01:17:46.499Z",
+    "approvals": [],
   },
```

Comment:
As approvals is not in the Bank Accounts model, we possibly need a description for this.